### DISCUSSION TITLE
老外代码已经适配xiaomi HD （r3d），不知lean大，可否适配支持？

### DISCUSSION THREAD

**Discussion by wgjwillis** [coolsnowwolf/lede#10269] [Ideas / 建议]
https://github.com/openwrt-xiaomi/openwrt/tree/xq-21.02.3
用他们的https://github.com/openwrt-xiaomi/builder 已经编译成功，如果能把r3d部分的适配代码整合过来，就完美了！！

**Comment by remittor** [comment]
**Released OpenWRT v23.05 for R3D**
https://github.com/openwrt-xiaomi/builder 
https://github.com/openwrt-xiaomi/openwrt/commits/xq-23.05.0
